packages/hardhat-etherscan/src/network/prober.ts

```diff
@@ -29,7 +29,7 @@ enum NetworkID {
   HECO = 128,
   HECO_TESTNET = 256,
   // Fantom mainnet
-  OPERA = 250
+  OPERA = 250,
```

Comment:
Unrelated to this PR, I don't know why the CI didn't catch this.

Reply:
This happened in a few other PRs. Not sure what's wrong.